Source/EventFlow.MsSql/EventFlow.MsSql.csproj

```diff
@@ -31,4 +31,8 @@
     <EmbeddedResource Include="EventStores\Scripts\0002 - Create eventdatamodel_list_type.sql" />
     <EmbeddedResource Include="SnapshotStores\Scripts\0001 - Create EventFlowSnapshots.sql" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.ComponentModel.Annotations" Version="4.4.0" />
```

Comment:
aren't these two package references required for netstandard? please add a condition